group.c

```diff
@@ -39,7 +40,8 @@ struct pv_group *pv_group_new(char *name, int timeout, plat_status_t status,
 	g = calloc(1, sizeof(struct pv_group));
 	if (g) {
 		g->name = strdup(name);
-		g->timeout = timeout;
+		g->timeout.limit = timeout;
+		g->timeout.started = false;
```

Comment:
I don't like this very much. why not putting started outside? in the end what was started is the group itself right? Maybe it's not even needed, as you can check if the timeout was initialized already?

Reply:
> I don't like this very much. why not putting started outside? in the end what was started is the group itself right?
Ok, it could be outside, it's only I think it could be better to group those elements, but that not a problem. 

> Maybe it's not even needed, as you can check if the timeout was initialized already?
I don't see any way to do that, if it's exists, maybe this is the opportunity to add a function or a field in `timer_state`

Reply:
If it's too much hassle to check if a timer has been initialized, just use the variable, but not inside of the struct. This is very similar to plat_status_t. So maybe do something similar to what we do there.